### ISSUE TITLE
Success Criterion 2.5.4 - Motion Actuation - Level A

### ISSUE THREAD

**Issue by JJdeGroot** [w3c/matf#40]
WCAG2ICT guidance: https://www.w3.org/TR/wcag2ict-22/#motion-actuation

Share your thoughts for applying to mobile apps as a comment below.

**Comment by julianmka** [comment]
Based on previous task force conversation, this SC can be applied to native mobile apps and mobile web with minimal or no deviation from WCAG2ICT.

### Proposal
In MATF's first draft of guidance, this SC's section will read as:

>This applies directly as written, and as described in [Intent from Understanding Success Criterion 2.5.4](https://www.w3.org/WAI/WCAG22/Understanding/motion-actuation.html#intent).

Please indicate your agreement with a thumbs up emoji 👍, or if you disagree, use the thumbs down emoji 👎 and elaborate in comments.

**Comment by jha11y** [comment]
<details>
<summary>Notes from 18 September, 2024 Meeting:</summary>

[Meeting Minutes 18 September](https://www.w3.org/2024/09/18-matf-minutes.html#t01)

quintinb: Please ensure I properly represent you - if you want to correct me I welcome the feedback!

Joe_Humbert: We may need to add some sections - he's going through it and will share it out with us on slack and email (action items in progress)

julianmka: This one is refreshingly straightforward as it seems to have been made with mobile in mind

[Accept WCAG SC as written in WCAG2ICT]

quintinb: +1

Illai: +1

Jamie: +1

Joe_Humbert: +1

julianmka: +1

AlainVagner: +1

Detlev: +1

Aashutosh: +1

Mick: +1

</details>

**Comment by JJdeGroot** [comment]
Closing this issue because the draft language has been accepted.
